src/main/java/delete_post/use_case/DeletePostResponseModel.java

```diff
@@ -0,0 +1,32 @@
+package delete_post.use_case;
+
+import entities.CurrentUser;
+import entities.Post;
+import entities.User;
+
+import java.util.List;
+
+public class DeletePostResponseModel {
+
+    private Post post;
+
+    public DeletePostResponseModel(Post post){
+        this.post = post;
```

Comment:
The response model also likely doesn't need the actual Post object - just the id.